packages/integration-sdk-cli/src/commands/options.ts

```diff
@@ -55,19 +97,90 @@ export function getSynchronizationJobSourceOptions(
       throw new Error(
         'Invalid configuration supplied. --source api requires --scope flag.',
       );
-    synchronizeOptions.scope = options.scope;
   } else if (
     !['integration-managed', 'integration-external'].includes(options.source)
   ) {
     throw new Error(
       `Invalid configuration supplied. --source must be one of: integration-managed, integration-external, api. Received: ${options.source}.`,
     );
-  } else if (options.integrationInstanceId) {
-    synchronizeOptions.integrationInstanceId = options.integrationInstanceId;
-  } else {
+  } else if (!options.integrationInstanceId) {
     throw new Error(
       'Invalid configuration supplied. --integrationInstanceId or --source api and --scope required.',
     );
   }
-  return synchronizeOptions;
+  return options;
+}
+
+/**
+ * Returns an object containing only `SyncOptions` properties. This helps to ensure other properties of `OptionValues` are not passed to the `SynchronizationJob`.
+ */
+export function getSyncOptions(options: OptionValues): SyncOptions {
+  return {
+    source: options.source,
+    scope: options.scope,
+    integrationInstanceId: options.integrationInstanceId,
+    uploadBatchSize: options.uploadBatchSize,
+    uploadRelationshipBatchSize: options.uploadRelationshipBatchSize,
+    skipFinalize: options.skipFinalize,
+  };
+}
+
+export interface ApiClientOptions {
+  development: boolean;
+  apiBaseUrl: string;
+  apiKey?: string;
+  account?: string;
+}
+
+export function addApiClientOptionsToCommand(command: Command): Command {
+  return command
+    .option(
+      '--api-base-url <url>',
+      'specify synchronization API base URL',
+      JUPITERONE_PROD_API_BASE_URL,
+    )
+    .option(
+      '-d, --development',
+      '"true" to target apps.dev.jupiterone.io (JUPITERONE_DEV environment variable)',
+      !!process.env.JUPITERONE_DEV,
+    )
+    .option(
+      '--account <account>',
```

Comment:
ditto: `accountId`

Reply:
I think we'll have to do this one in another PR that allows for using both and deprecating one, since as I understand it, this was an option already for some commands.